services/monitoring/grafana/terraform/datasources.tf

```diff
@@ -0,0 +1,25 @@
+
+resource "grafana_data_source" "prometheusfederation" {
+  type               = "prometheus"
+  name               = "prometheus-federation"
+  url                = var.prometheus_federation_url
+  basic_auth_enabled = false
+  is_default         = true
+}
+
+resource "grafana_data_source" "prometheuscatchall" {
+  type               = "prometheus"
+  name               = "prometheus-catchall"
+  url                = var.prometheus_catchall_url
+  basic_auth_enabled = false
+  is_default         = false
+  uid                = "RmZEr52nz"
```

Comment:
This "could" have been a password (the academic proepr term is "high entropy string" I guess). I think we should have some sort of test (pre-commit?) that asserts that no high-entropy strings are commited in this repo, as it is public on github, and we might leak secrets, and rotating secrets is a lot of annoying work, so I'd like that not to happen :D